segments/suite_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+package segments
+
+import (
+	"bufio"
+	"gopkg.in/check.v1"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func Test(t *testing.T) { check.TestingT(t) }
+
+type S struct {
+	sampleBaseSegment string
+}
+
+var _ = check.Suite(&S{})
+
+func (s *S) SetUpSuite(c *check.C) {
+	f, err := os.Open(filepath.Join("../testdata", "base_segment.dat"))
```

Comment:
To be entirely proper we'd use `filepath.Join("..", "testdata", "base_segment.dat")`

Reply:
fixed